cmd/gensign/main.go

```diff
@@ -115,8 +121,8 @@ func main() {
 		if gensign.IsErrorOfType(err, gensign.Panic) {
 			// gensign will return debug stack in err when panic.
 			// We do not want it to be printed to os.Stderr since it will also go to user's console.
-			log.Logger = log.Output(file)
+			log.Logger = fileLogger
 		}
-		log.Fatal().Str(logkey.TransIDField, reqParam.TransID).Err(err).Msg("failed to run gensign")
+		log.Error().Str(logkey.TransIDField, reqParam.TransID).Err(err).Msg("failed to run gensign")
```

Comment:
Change to `Error` to prevent from `os.Exit()` as it would not invoke defer functions.